packages/engine/src/scene/functions/loaders/AudioFunctions.ts

```diff
@@ -115,14 +113,13 @@ export const updateAudio: ComponentUpdateFunction = async (entity: Entity, prope
 
   if (properties.audioSource) {
     try {
-      const { url, contentType } = await resolveMedia(component.audioSource)
```

Comment:
By removing `resolveMedia` we are relying on file extension to determine extension instead of content type. This can cause problems for URL sources with no extensions. If this is intentional then avoid this.

Reply:
yeah this was something i realised, since all files being uploaded in the editor will have a file extension, i'm not too worried, it's pretty standard now. it also enforces ensuring file extensions correspond to mime types, which is what we want.